File: 1344-AngleBetweenHandsOfAClock/1344-AngleBetweenHandsOfAClock.cpp
// Last updated: 9/14/2026, 11:56:01 PM
class Solution {
public:
    double angleClock(int hour, int minutes) {
        double hourangle= (hour%12)*30 +minutes*0.5;
        double minuteangle= minutes*6;
        double diff= abs(hourangle- minuteangle);
        return min(diff, 360-diff);
    }
};

File: 1344-AngleBetweenHandsOfAClock/1344-AngleBetweenHandsOfAClock.py
# Last updated: 9/14/2026, 11:55:55 PM
class Solution(object):
    def angleClock(self, hour, minutes):
        """
        :type hour: int
        :type minutes: int
        :rtype: float
        """
        hour_angle = (hour % 12) * 30 + minutes * 0.5
        minute_angle = minutes * 6
        diff = abs(hour_angle - minute_angle)
        return min(diff, 360 - diff)